### DISCUSSION TITLE
M1 and M2 chips

### DISCUSSION THREAD

**Discussion by WilkAndy** [idaholab/moose#21332] [Q&A Getting Started] [answered]
I'm thinking of buying a new computer.  Does MOOSE work on Apple M1 and M2 chips?

**Comment by WilkAndy** [comment]
Refs #20811

**Comment by GiudGiud** [comment, marked as answer]
Hello

Yes it does for M1. There is a working conda distribution for it too. We will have a M2 in about a month to try.
I think there is still a few modules failures on M1 see this PR https://github.com/idaholab/moose/pull/20349

Guillaume

**Comment by joe61vette** [comment]
Hi:  what would also help those of us considering a new Mac purchase would be some guidance on memory.  As the new unified memory works somewhat differently, less is supposedly needed.  Is that true for Moose based apps? If anyone who is currently running Moose on an M1 could post (for example) memory usage vs DOFs that would really help.

I know the normal guidance is to buy all the memory you can afford, but with the memory being non-upgradeable and coming at a premium price some advice would help.

Thanks,
   Joe Kelly

**Reply by loganharbour** [reply]
There won't be much of a difference. You should go with what satisfies your current needs.
